test_scripts/API/SetAudioStreamingIndicator/019_ATF_N_TC_SetAudioStreamingIndicator_param_wrongtype_navi_app_SDL_replies_INVALID_DATA.lua

```diff
@@ -0,0 +1,75 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SetAudioStreamingIndicator] SDL must transfer request from mobile app to HMI in case no any failures
+-- [GeneralResultCodes] INVALID_DATA wrong type
+-- [MOBILE_API] SetAudioStreamingIndicator
+-- [HMI_API] [MOBILE_API] AudioStreamingIndicator enum
+-- [PolicyTable] SetAudioStreamingIndicator RPC
+--
+-- Description:
+-- In case media app sends the invalid SetAudioStreamingIndicator_request to SDL
+-- and this request is allowed by Policies
+-- SDL must NOT transfer SetAudioStreamingIndicator_request to HMI
+-- SDL must respond with result code INVALID_DATA
+--
+-- 1. Used preconditions
+-- Allow SetAudioStreamingIndicator RPC by policy
+-- Register and activate navi application
+--
+-- 2. Performed steps
+-- Send SetAudioStreamingIndicator(audioStreamingIndicator = 123),
+-- audioStreamingIndicator has wrong type: integer
+--
+-- Expected result:
+-- SDL->mobile: SetAudioStreamingIndicator_response(INVALID_DATA, success:false)
+-- SDL must NOT transfer this SetAudioStreamingIndicator_request to HMI
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.application1.registerAppInterfaceParams.appHMIType = {"NAVIGATION"}
```

Comment:
@dboltovskyi, yes, you are correct that  appHMIType in config.application1.registerAppInterfaceParams is NAVIGATION, but with this declaration, I guarantee that in case of change config.application1.registerAppInterfaceParams.appHMIType to some other type, script will be run for navigation application.

According to "isMediaApplication": based on clarification: "Can you clarify RAI paramenters for navi and voice-com application?" isMediaApplication can have any value(true/false).
Regarding this verification detailed check is done inhttps://github.com/smartdevicelink/sdl_atf_test_scripts/pull/940
Note: the same is for voice-com